.github/workflows/release_workflow.yml

```
@@ -0,0 +1,105 @@
+        uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Get version from setup.py
+        id: get_version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)